### ISSUE TITLE
Parrot Bind Test Failing

### ISSUE THREAD

**Issue by dthain** [cooperative-computing-lab/cctools#1571]
It seems that the parrot bind test failed recently, though it's not immediately clear what commit caused it:
http://ccl.cse.nd.edu/software/autobuild/

The test fails on an attempt to bind a Unix domain socket.
Reproduce by logging into cclvm04.crc.nd.edu, building master and running this:
```
% cd parrot/test
% ./TR_parrot_bind.sh prepare
% ./TR_parrot_bind.sh run
[<stdin>:35] bind(master, (struct sockaddr *)&addr, sizeof(addr)): Operation not permitted
...
```
@trshaffer please take a look at this, since you most recently looked at Unix domain sockets in parrot.

**Comment by trshaffer** [comment]
I get this error even if I'm not running under Parrot, so it appears to be a problem with the test itself. I just put in a PR to resolve it

**Comment by dthain** [comment]
Ok, that's a relief

**Comment by dthain** [comment]
Resolved by #1574
